src/test/java/com/elasticbox/jenkins/tests/PullRequestLifecycleManagementTest.java

```diff
@@ -45,6 +45,7 @@
     @Test
     public void testPullRequestLifecycleManagement() throws Exception {
 
+        long NEXTBUILD_TIMEOUT = 120; // before was 30
```

Comment:
Please, remove comment. Are 2 mins really required? Maybe we can try to lower it a bit...